### PR TITLE
Adding precision about which letter case to use for controller names …

### PR DESCRIPTION
**Pull request by simonjamain**
…in routing

Many people (including myself) encounter an error when having multiple words controller names and trying to put camelCase in their routes

### Summary

Provide a general description of the code changes in your pull
request... were there any bugs you had fixed? If so, mention them. If
these bugs have open GitHub issues, be sure to tag them here as well,
to keep the conversation linked together.

### Other Information

If there's anything else that's important and relevant to your pull
request, mention that information here. This could include
benchmarks, or other information.

If you are updating any of the CHANGELOG files or are asked to update the
CHANGELOG files by reviewers, please add the CHANGELOG entry at the top of the file.

Finally, if your pull request affects documentation or any non-code
changes, guidelines for those changes are [available
here](http://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#contributing-to-the-rails-documentation)

Thanks for contributing to Rails!

### CODE REVIEW

File: guides/source/routing.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ get '/patients/:id', to: 'patients#show'
 
 the request is dispatched to the `patients` controller's `show` action with `{ id: '17' }` in `params`.
 
+NOTE: rails uses snake_case for controller names here, if you have a multiple word controller like `MonsterTrucksController`, you want to use `monster_trucks#show` for example.
```
Comment (eileencodes) [inline guides/source/routing.md]: Can you capitalize Rails?

**Review by eileencodes — CHANGES_REQUESTED** [review]
Thanks for the PR! I requested one change.

File: guides/source/routing.md (hunk 1 above)
Comment (simonjamain) [reply guides/source/routing.md]: sure, once I figure out how to do that, its my first PR ^^

Comment (eileencodes) [reply guides/source/routing.md]: Did you make this change locally or in the GitHub UI?

**Comment by simonjamain** [comment]
github UI

2018-04-27 20:02 GMT+02:00 Eileen M. Uchitelle <notifications@github.com>:

File: guides/source/routing.md (hunk 1 above)
Comment (simonjamain) [reply guides/source/routing.md]: github UI

Comment (eileencodes) [reply guides/source/routing.md]: Ok go to the "Files changed" tab, then click the "pencil". Make the change and add a new commit.

**Comment by simonjamain** [comment]
Thank you for your time @eileencodes

**Review by eileencodes — APPROVED** [review]

**Comment by eileencodes** [comment]
Thanks for the quick fix @simonjamain and congrats on your first PR! 🎉

**Pull request merged**